src/components/Carousel/index.tsx

```diff
@@ -611,12 +607,12 @@ export default class Carousel extends React.Component<CarouselProps, CarouselSta
         return null;
     };
 
-    renderItems(isClone?: boolean) {
+    renderItems(isClone?: boolean): React.ReactElement[] {
         if (!this.props.children) {
             return [];
         }
 
-        return Children.map(this.props.children, (item, index) => {
+        return Children.toArray(this.props.children).map((item, index) => {
```

Comment:
Using `.toArray(...).map` instead as it will remove falsy children, which is both more correct behaviour, and helped with typings (e.g. handle conditional rendering properly)